src/main/java/com/datatheorem/mobileappsecurity/jenkins/plugin/SendBuildToDataTheoremPublisher.java

```diff
@@ -375,7 +399,8 @@ public void setApplicationCredentialComments(String applicationCredentialComment
             "sendBuildDirectlyFromRemote",
             "applicationCredentialUsername",
             "applicationCredentialPassword",
-            "applicationCredentialComments"
+            "applicationCredentialComments",
+            "releaseType"
```

Comment:
In general, a trick to reduce diffs and increase consistency can be to add a comma "," at the end of each of the items in this kind of "list", and I don't think there's any restriction against that in Java.